src/main/java/com/kuljava/swiatwsi/rawmaterials/RawMaterials.java

```diff
@@ -0,0 +1,34 @@
+package com.kuljava.swiatwsi.rawmaterials;
+
+import com.kuljava.swiatwsi.world.Village;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "raw_materials")
+@Getter
+@Setter
+@ToString
```

Comment:
Where are You using toString? Is it neccessary?

Reply:
No, it isn't neccessary

Reply:
See comment about setters